hr_employee_spouse/data/hr_employee_spouse_data.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+      <record id="res_partner_category_spouse" model="res.partner.category">
```

Comment:
the record should be forcecreate I guess? Or if not, your code must cope with the reference being empty